src/platform/packages/shared/kbn-rule-data-utils/src/rule_types/o11y_rules.ts

```diff
@@ -42,6 +42,12 @@ export const SYNTHETICS_RULE_TYPE_IDS = [SYNTHETICS_STATUS_RULE, SYNTHETICS_TLS_
 export const SLO_BURN_RATE_RULE_TYPE_ID = 'slo.rules.burnRate';
 export const SLO_RULE_TYPE_IDS = [SLO_BURN_RATE_RULE_TYPE_ID];
 
+/**
+ * ESQL rule types
+ */
+export const ESQL_RULE_TYPE_ID = 'streams.rules.esql';
```

Comment:
```suggestion
export const STREAM_ESQL_RULE_TYPE_ID = 'streams.rules.esql';
```
nit: Just to avoid the confusion with the stack ES rule type.